compute/admin_guide/vulnerability_management/malware_scanning.adoc

```diff
@@ -13,7 +14,7 @@ Windows containers are not supported.
 [.task]
 === Detecting malware
 
-When Prisma Cloud detects malware in an image, it logs the vulnerability in the image scan report.
+The image scanner looks for malware in binaries in the image layers, including the base layer. When Prisma Cloud detects malware in an image, it logs the vulnerability in the image scan report.
```

Comment:
Actually it's not exactly a vulnerability, when it finds malware it is reported as a compliance issue.